examples/kubernetes-cluster/variables.tf

```diff
@@ -7,29 +7,20 @@ variable "aws_region" {
 variable "service_cidr" {
   description = "(Optional) The Kubernetes service CIDR."
   type        = string
-  default     = "172.16.0.0/13"
+  default     = "10.96.0.0/12"
 }
 
 variable "cluster_cidr" {
   description = "(Optional) The Kubernetes cluster CIDR."
   type        = string
-  default     = "172.24.0.0/13"
+  default     = "10.244.0.0/16"
```

Comment:
Maybe align with VPC CIDR will be better

Reply:
done. https://github.com/getamis/vishwakarma/blob/ba0396e4d4f5161f3f6c595ee1d3976e32de081a/examples/kubernetes-cluster/main.tf#L24-L26